Gemfile.lock

```diff
@@ -2,7 +2,7 @@ PATH
   remote: .
   specs:
     rubocop-shopify (1.0.4)
-      rubocop (>= 0.85, < 0.89)
+      rubocop (= 0.89)
```

Comment:
You might need to run `bundle install` to fix this.

Reply:
I don't see anything wrong?

Reply:
ah! Got it.